AUTHORS

```diff
@@ -1,8 +1,12 @@
 Darron Froese <dfroese@salesforce.com>
 Eric Hole <eric.hole@gmail.com>
+Igor Vuk <parcijala@gmail.com>
 Joe Beda <joe.github@bedafamily.com>
 Joshua Carp <jm.carp@gmail.com>
+@Kris__Nova <kris@nivenly.com>
 Kris Nova <kris@nivenly.com>
+Marko Mudrinić <mudrinic.mare@gmail.com>
 Michael Hausenblas <michael.hausenblas@gmail.com>
 Paul Czarkowski <username.taken@gmail.com>
 Ted Wood <ted@mailchimp.com>
+Ted Wood <ted@xy0.org>
```

Comment:
...I think I see what happened: https://stackoverflow.com/a/40552998/6177740

I'm working on cleaning this up and will submit a PR when I've done it. Sorry for the mess.